### DISCUSSION TITLE
Selecting columns with a regular expression

### DISCUSSION THREAD

**Discussion by jllipatz** [duckdb/duckdb#5331] [General]
Documentation on duckdb.org suggests that there is a way to select columns using a common property on their name. This should be using the keyword 'COLUMNS' but I am not able to find the correct syntax. Is there some example somewhere? 

Thanks

**Comment by Mytherin** [comment]
The example documentation is [here](https://duckdb.org/docs/sql/expressions/star). Note that this is not possible on the current release (`0.5.1`) but will be possible on the latest release (`0.6.0`) which we are releasing later today.

**Reply by jllipatz** [reply]
Thanks, everything is being clear now!
I shall wait 0.6
